src/controller/userController.js

```diff
@@ -379,18 +378,66 @@ class UserController {
         }
       );
       if (userUpdated[0] === 1) {
-        return res.status(200).send({
+        return res.status(200).json({
           message: 'User role was updated successfully',
           success: true
         });
       }
     } catch (error) {
-      res.status(500).send({
+      res.status(500).json({
         message: 'Internal server error',
         success: false,
       });
     }
   }
+
+  /**
+   * User Login
+   * Route: POST: auth/login
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @return {res} res - Response object
+   * @memberof UserController
+   */
+  static async userLogin(req, res) {
+    const { email, password } = req.body;
+    try {
+      const userFound = await Users.findOne({ where: { email, } });
```

Comment:
```suggestion
      const userFound = await Users.findOne({ where: { email, isVerified: true} });
```

Reply:
Thank you for point this out. I will add it.

Reply:
Checking if the user is verified has been added. Thank you once again.